test/expected/plan_expand_hypertable-11.out

```diff
@@ -997,11 +997,11 @@ SELECT * FROM cte ORDER BY value;
          ->  Index Only Scan Backward using _hyper_6_163_chunk_metrics_timestamptz_time_idx on _hyper_6_163_chunk m1_4
          ->  Index Only Scan Backward using _hyper_6_164_chunk_metrics_timestamptz_time_idx on _hyper_6_164_chunk m1_5
    ->  Materialize
-         ->  Custom Scan (ChunkAppend) on metrics_timestamptz m2
-               Order: m2."time"
-               ->  Index Only Scan Backward using _hyper_6_160_chunk_metrics_timestamptz_time_idx on _hyper_6_160_chunk m2_1
+         ->  Merge Append
```

Comment:
why did this revert back to merge append? Also applies below.

Reply:
fixed